content/pages/burials-and-memorials/application/530.md

```diff
@@ -15,8 +15,9 @@ layout: page-react.html
   <div class="section">
     <div id="react-root">
       <div class="loading-message">
-        <h3>Please wait while we load the application for you.</h3>
-        <img src="/img/preloader-primary-darkest.gif" alt="Loading">
+        <div class="loading-indicator-container">
+          <div class="loading-indicator" role="progressbar" aria-valuetext="Please wait while we load the application for you." tabIndex="0"></div> Please wait while we load the application for you.
```

Comment:
Something to note with this, the last time I checked the new spinner svg was twice the size of the older gif spinner. That could mean there's a longer delay for seeing the spinner itself.

Reply:
That makes sense. I think that may occur on the first request but not subsequent because it'll be in cache. If it becomes a problem at all maybe we can start preloading it with JS or somewhere on the page.